.eslintrc.js

```diff
@@ -46,6 +46,7 @@ module.exports = {
     'plugin:eslint-comments/recommended',
     'plugin:import/errors',
     'plugin:import/warnings',
+    'plugin:import/typescript',
```

Comment:
This was needed to solve the problem where `.js` files were reporting that they couldn't find their dependencies that had been moved to `.ts`.